GOVERNANCE.rst

```diff
@@ -335,16 +346,51 @@ sktime uses a “consensus seeking” process for making decisions. The
 community tries to find a resolution that has no open objections among
 core developers.
 
--  To accept proposed changes, we require approval by one core developer
-   (lazy consensus) and no rejection by a core developer (veto right).
+-  For a proposed changes to be approved via lazy consensus, one of the following
+   two conditions must be satisfied:
+    - approval by at least one core developer and no rejection by a core developer (veto right).
+      The approval required for this condition must be by a core developer different from a proposer.
+    - approval in the regular core developer meeting as part of the agenda.
+      This can happen only if no approvals or rejections are expresesd on a PR, see below.
+-  For a proposed change to be rejected via lazy consensus, one of the following two conditions
+   must be satisfied:
+    - rejection by at least one core developer, and no acceptance by a core developer.
+    - rejection in the regular core developer meeting as part of the agenda.
+      This can happen only if no approvals or rejections are expresesd on a PR, see below.
+-  Formal statements on proposed changes must be located with the proposal on GitHub,
+   usually in the same pull request that contains the proposal, or the related issue.
+   Formal statements in this sense are approvals/rejections, and statements
+   about whether the proposal should be considered a "major change", as below.
 -  Approvals and rejections can be expressed as +1 and -1 comments,
-   respectively.
--  Core developers are expected to give reasonable time to others to
-   give their opinion on the pull request if they’re not confident
-   others would agree.
--  More important changes that impact the full project require a more
+   respectively, or any written comments
+   containing "I formally approve/reject" in reference to the proposal.
+   If the proposed change has the form of a PR, approval must be in the form of a GitHub PR approval.
+-  Major changes to the project require a more
    detailed analysis and a consensus that is both explicit and informed.
-   These changes require an :ref:`steps`.
+   A major change always require a worked out :ref:`steps`.
+   A change is major in the above sense
+   if at least three core developers explicitly state that the proposal is a major change
+   and requires especial consideration, in the location.
```

Comment:
I think some changes are major changes even if less than three core devs explicitly state that on the PR/issue. I'd say we leave out the definition of what counts as "major" here and resolve that through discussion if unclear. 
```suggestion
```

Reply:
well, but if we don't define what is "major", we cannot make "major" a condition in the rules, since it is undefined and arbitrary otherwise. It appears as a condition for PR that require a STEP.

So either we remove both the requirement of STEPS too, or we leave both things in.

Reply:
deferred to the table